common/src/main/java/com/wynntils/features/commands/CommandAliasesFeature.java

```diff
@@ -35,7 +35,7 @@ public class CommandAliasesFeature extends UserFeature {
             new CommandAlias("guild territory", List.of("gu t", "gu terr", "guild t", "guild terr"))));
 
     @TypeOverride
-    private final Type aliasesType = new TypeToken<List<CommandAlias>>() {}.getType();
+    private final Type aliasesType = new TypeToken<ArrayList<CommandAlias>>() {}.getType();
```

Comment:
I had a months old branch cleaning these up, never got around finishing other parts of it. Thanks!